pkg/services/govmomi/vcenter/clone.go

```diff
@@ -46,7 +46,7 @@ const (
 // the virtual machine to be created on the vCenter, which can be resolved by waiting on the task reference stored
 // in VMContext.VSphereVM.Status.TaskRef.
 //
-//nolint:gocognit,gocyclo
+//nolint:gocognit
```

Comment:
For me locally we can also get rid of this

Reply:
Neither of these is enabled in our config

Reply:
Missed this.